### ISSUE TITLE
react-native-config does not read variables from env file

### ISSUE THREAD

**Issue by satishmane** [react-native-config/react-native-config#129]
Hi,

"react": "16.0.0-alpha.12",
 "react-native": "0.45.1"
"react-native-config": "0.3.0",

Performed below steps

1.Set the value for Info.plist Preprocessor Prefix File ${SYMROOT}/Release$(EFFECTIVE_PLATFORM_NAME)/GeneratedInfoPlistDotEnv.h
2.Set Info.plist Other Preprocessor Flags to -traditional
3. Set Preprocess Info.plist File to Yes
4. Created my scheme and entered my env file as echo ".env.dev" >> /tmp/envfile in pre-actions and also did the same under archive pre-actions
5. Selected my scheme.
6. Build->Archive->export etc
7. Deployed .ipa file on mobile
8. It does not show env specific changes to screen

Out of ten times, it worked couple of times. Not sure what is causing this problem. I closed xcode and restarted, clean build and repeated step 5 to 7. No luck

I can see my .env.dev file being copied to /tmp/envfile. But my env variables are not read by config package.

I am fed up with react-native-config. Any help would be appreciated

**Comment by CoreyAR** [comment]
You may want to check what is being copied to `/tmp/envfile`. I had to remove the quotation marks from the example in the README because it was copying the string ".env.staging" and not the contents of the file. `echo .env.development > /tmp/envfile`

**Comment by satishmane** [comment]
Yes, I do checked contents of files anf they are correct. But still it does not make those content available in config object which is imported from react-native-config.

Sent from my iPhone

On Aug 1, 2017, at 10:45 PM, Corey Rice <notifications@github.com<mailto:notifications@github.com>> wrote:


You may want to check what is being copied to /tmp/envfile. I had to remove the quotation marks from the example in the README because it was copying the string ".env.staging" and not the contents of the file. echo .env.development > /tmp/envfile

**Comment by damathryx** [comment]
same problem @satishmane  have you solved this?

**Comment by satishmane** [comment]
Yes, I resolved it. Basically problem was in project.pbxproj. There are multiple occurrences of below variable in pbxproj. At some places, path was different. I made it like as below everywhere. Created schemes and added echo statement as per documentation.

INFOPLIST_PREFIX_HEADER = "$(BUILD_DIR)/GeneratedInfoPlistDotEnv.h";


________________________________

**Comment by canhnht** [comment]
I had the same issue when using react-native-config 0.10 in iOS. I downgrade to 0.8.1 and it works okay.

**Comment by jakabjack** [comment]
@canhnht that's right, it looks like react-native-config 0.10 has some issues on iOS
